.github/workflows/docker-build.yaml

```diff
@@ -69,10 +71,14 @@ jobs:
               docker/Dockerfile.base
             sed -i 's/balenalib\/raspberrypi3/balenalib\/raspberry-pi2/' \
               docker/Dockerfile.viewer
+            sed -i 's/\barmv7hf\b/rpi/' \
+              docker/Dockerfile.wifi-connect
             echo 'TARGET_PLATFORM=linux/arm/v6' >> $GITHUB_ENV
 
           # We use the same target for Pi 3 and Pi 4 for now.
           else
+            sed -i 's/\bBALENA_ARCH\b/armv7hf/' \
```

Comment:
This shouldn't be necessary if we're using the regular base image.

Reply:
Got it. Will remove this one.